pages/_app.jsx

```diff
@@ -31,39 +61,27 @@ export async function getStaticProps({ res }) {
       `script-src 'self' *.google-analytics.com ${
         process.env.NODE_ENV !== 'production' ? "'unsafe-inline'" : ''
       }`,
-      `style-src 'self' ${
-        process.env.NODE_ENV !== 'production' ? "'unsafe-inline'" : ''
-      }`,
+      "style-src 'self' 'unsafe-inline'",
```

Comment:
We should disallow inline styles once this is solved https://github.com/vercel/next.js/issues/17445